Mathlib/MeasureTheory/Measure/Lebesgue/Integral.lean

```diff
@@ -74,6 +74,23 @@ mathlib's conventions for integrals over finite intervals (see `intervalIntegral
 of finite integrals, see `intervalIntegral.integral_comp_neg`.
 -/
 
+theorem MeasureTheory.IntegrableOn.comp_neg_Iic {E : Type*} [NormedAddCommGroup E]
+    {c : ℝ} {f : ℝ → E} (hf : IntegrableOn f (Ioi (-c))) :
+    IntegrableOn (fun x => f (-x)) (Iic c) := by
```

Comment:
Please prove the more general version which is analogous to: [MeasureTheory.Integrable.comp_neg](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Group/Integral.html#MeasureTheory.Integrable.comp_neg). It should be `(hf : IntegrableOn f s) : IntegrableOn (fun x ↦ f (-x)) -s)` (and you'll probably need `open scoped Pointwise` for the `-s` to make sense).

You can then prove both these lemmas as special cases of that one.

Reply:
Thanks for the pointer. I think I get it now.